src/main/java/org/gridsuite/modification/server/repositories/NetworkModificationRepository.java

```diff
@@ -85,6 +85,18 @@ public UUID createNetworkCompositeModification(@NonNull List<UUID> modificationU
         return modificationRepository.save(compositeEntity).getId();
     }
 
+    public void updateCompositeModification(@NonNull UUID compositeUuid, @NonNull List<UUID> modificationUuids) {
+        CompositeModificationEntity compositeEntity = (CompositeModificationEntity) modificationRepository.findById(compositeUuid)
```

Comment:
For safety, maybe check if the entity is a composite modification entity before casting it ...